custom/nic_compliance/utils.py

```diff
@@ -66,35 +72,36 @@ def verify_password(password, encoded_password):
     return PASSWORD_HASHER.verify(password, encoded_password)
 
 
-def obfuscated_passwords_redis_key_for_user(username):
-    return REDIS_LOGIN_ATTEMPTS_LIST_PREFIX + username
+def obfuscated_password_redis_key_for_user(username, obfuscated_password):
+    return '_'.join([REDIS_LOGIN_ATTEMPTS_LIST_PREFIX, username, obfuscated_password])
```

Comment:
can we make this prefix shorter to reduce memory requirement

Reply:
i updated `REDIS_LOGIN_ATTEMPTS_LIST_PREFIX` to be just `la$$`. Rest is username and obfuscated password.

We need username and obfuscated password to identify this uniquely. Can also do md5 hash of the username & password and use that as the key instead

for example:
```
In [16]: username
Out[16]: 'mkangia@dimagi.com'

In [17]: obf_password
Out[17]: 'sha1$SUXeJGr1jWhM$902d24b15a1cb63f5b6c989d53517ed76764c7ef'

In [18]: REDIS_LOGIN_ATTEMPTS_LIST_PREFIX  + hashlib.md5("%s%s"% (username, obf_password)).hexdigest()
Out[18]: '$$la901270bcbe77d5ce914c60592a201d38'
```